doc/node.1

```diff
@@ -349,6 +349,10 @@ and
 .It Fl -trace-events-enabled
 Enable the collection of trace event tracing information.
 .
+.It Fl -trace-exit
+Prints a stack trace whenever an environment is been exited proactively, i.e.
```

Comment:
See above; the typo has been made here too.

In addition, I recommend putting `\&` after `i.e.` to stop `troff` mistaking it as the end of a sentence. Doing so will stop an uncomfortable gap being inserted due to paragraph justification:

~~~
exited proactively, i.e.  invoking  # Without \&
exited proactively, i.e. invoking   # With \&
~~~

`\&` is Roff's _“do nothing”_ character, which is helpful to suppress the behaviour of constructs that have syntactic meaning (for example, starting a line with a dot, which is Roff's way of identifying formatting instructions).